docs/5-operator-manual/90-supported-oses.md

```diff
@@ -10,66 +10,12 @@ Check the table below to understand which distros which we test against & if the
 <!-- TODO: @JPERRY this support matrix could probably just go into a FAQ?  -->
 ## Support Matrix
 
-|OS             |VM_ID      |Notes|
-|---            |---        |---|
-|RHEL 7         |rhel7      ||
-|RHEL 8         |rhel8      ||
-|CentOS 7       |centos7    ||
-|CentOS 8       |centos8    ||
-|Ubuntu 20.04   |ubuntu     ||
-|Debian 11      |debian     ||
-|Rocky 8.4      |rocky      ||
-
-<!-- TODO: @JPERRY Is any of the content below this comment actually useful? -->
-## Demo Environments
-
-We support running an instance of Zarf _inside a local VM_ (of any of the [supported OSes](#support-matrix)) for test & demonstration purposes.
-
-> _**Take note**_
->
-> Run the following commands from  _**the project root directory**_.
-
-### Startup
-
-To get a VM running, it's as easy as running a single command:
-
-``` bash
-make vm-init OS=[VM_ID]     # e.g. make vm-init OS=ubuntu
-```
-
-> _**Warning!**_
->
-> Besure to pass a VM_ID or you'll start a VM instance for _every one of the supported OS types_. Yikes!
-
-### Work in the VM
-
-To connect into the VM instance you just started, run:
-
-``` bash
-vagrant ssh [VM_ID]         # e.g. vagrant ssh ubuntu
-```
-
-Once connected, you can work with your mounted-from-the-host copy of Zarf like so:
-
-``` bash
-sudo su                     # escalate permissions (to "root" user)
-cd /opt/zarf                # access Zarf
-./zarf help
-```
-
-When you're done with the VM, you can exit back to the host terminal by running:
-
-``` bash
-exit                        # de-escalate permissions (back to "vagrant" user)
-exit                        # exits VM shel & drops you back on the host
-```
-
-### Shutdown
-
-Closing out the demo environment is _also_ a single command:
-
-``` bash
-make vm-destroy
-```
-
-This will shutdown & destroy _all_ the demo VM instances it can find.  Easy-peasy&mdash;nice and clean.
+| OS           | VM_ID   | Notes |
+| ------------ | ------- | ----- |
+| RHEL 7       | rhel7   |       |
+| RHEL 8       | rhel8   |       |
+| CentOS 7     | centos7 |       |
+| CentOS 8     | centos8 |       |
+| Ubuntu 20.04 | ubuntu  |       |
+| Debian 11    | debian  |       |
+| Rocky 8.4    | rocky   |       |
```

Comment:
This should probably just be an FAQ answer, and likely shouldn't be a table since "supported OS" is very fluid depending on what features you are talking about.  In general we support Linux, macOS, and Windows as clients and Linux that uses systemd as an init system for our Appliance Mode.

Reply:
removing entire file. Will he re-created when testing is stabilized.